modules/react/button/lib/BaseButton.tsx

```diff
@@ -169,7 +169,7 @@ export const buttonStencil = createStencil({
       opacity: cssVar(buttonColorPropVars.default.opacity, cssVar(opacity, system.opacity.full)),
     },
     // Focus Styles
-    '&:focus-visible, &.focus': {
+    '&:focus, &.focus': {
```

Comment:
is this a breaking change? I thought we moved away from `:focus` to `focus-visibile`.